packages/nextjs/scaffold.config.ts

```diff
@@ -6,7 +6,7 @@ type ScaffoldConfig = {
   alchemyApiKey: string;
 };
 
-const scaffoldConfig: ScaffoldConfig = {
+const scaffoldConfig = {
   // The network where your DApp lives in
   targetNetwork: chains.hardhat,
```

Comment:
That would probably widen the type here, which is what we want to avoid. Otherwise the pr that I'm working on wouldn't work. We want the type to be exact, not `number`